### PR TITLE
Release 1.0 beta

### PR DESCRIPTION
**Pull request by gianluz**

### CODE REVIEW

**Comment by gianluz** [comment]
We probably want to merge all pending PR first, this will be merged as last

File: scripts/install.sh [hunk 1]
```diff
@@ -38,7 +38,7 @@ if ! [[ -x "$(command -v gradle)" ]]; then
     rm -rf gradle.zip
 fi
 
-LATEST_STABLE_VERSION=0.7.1
+LATEST_STABLE_VERSION=1.0-beta
```
Comment (gianluz) [inline scripts/install.sh:41]: not sure about this.. do we want to start using the beta here?

File: danger-kotlin-library/version.gradle [hunk 2]
```diff
@@ -1,2 +1,2 @@
 group 'systems.danger'
-version '0.7.1'
+version '1.0-beta'
```
Comment (f-meloni) [inline danger-kotlin-library/version.gradle:2]: It should be `1.0.0-beta` they should be three according to https://semver.org/

```
<valid semver> ::= <version core>
                 | <version core> "-" <pre-release>
                 | <version core> "+" <build>
                 | <version core> "-" <pre-release> "+" <build>

<version core> ::= <major> "." <minor> "." <patch>
```

Comment (gianluz) [reply danger-kotlin-library/version.gradle]: done in 0408ef2

**Review by f-meloni — APPROVED** [review]

**Pull request merged**